joint_names_order:
  - "joint_1"
  - "joint_2"
  - "joint_3"
  - "joint_4"
  - "joint_5"
  - "joint_6"

joint_max_vel:
  - 0.383012504350156  # Joint velocity value for joint_1
  - 0.37256670877697   # Joint velocity value for joint_2
  - 0.68942250783028   # Joint velocity value for joint_3
  - 0.602189591308619  # Joint velocity value for joint_4
  - 0.602189591308619  # Joint velocity value for joint_5
  - 1.05386470893922   # Joint velocity value for joint_6

joint_pos_limits_lower:
  - -2.617994  # Lower limit for joint_1
  - -1.047198  # Lower limit for joint_2
  - 0.3141593  # Lower limit for joint_3
  - -2.96706   # Lower limit for joint_4
  - -1.658063  # Lower limit for joint_5
  - -2.96706   # Lower limit for joint_6

joint_pos_limits_upper:
  - 2.617994  # Upper limit for j1
  - 2.617994  # Upper limit for j2
  - 2.443461  # Upper limit for j3
  - 2.96706  # Upper limit for j4
  - 2.356194  # Upper limit for j5
  - 2.96706  # Upper limit for j6

move_group_name: "arm"
joint_trajectory_controller_command_topic: "/cobotta/arm_controller/command"
cartesian_compliance_controller_command_topic: "/target_frame"
cartesian_motion_controller_command_topic: "/target_frame"
ee_link: "pestle_tip"
joint_states_topic: "/joint_states"
controller_type: "joint_trajectory_controller" # "joint_trajectory_controller" or "cartesian_compliance_controller"
wrench_topic: "/wrench/filtered"
gello_mode: "unilateral" # "unilateral" or "bilateral"
use_gripper: False
use_FT_sensor: False
gello_port: "/dev/serial/by-id/usb-FTDI_USB__-__Serial_Converter_FT88YXAT-if00-port0"

gello_start_joints:
  - 0.0  # joint_1
  - 0.0  # joint_2
  - !degrees 90.0  # joint_3
  - 0.0  # joint_4
  - !degrees 90.0  # joint_5
  - 0.0  # joint_6

stall_torque: 0.52
stall_current: 1.5
torque_rate:
  - 0.003
  - 0.003
  - 0.003
  - 0.003
  - 0.003
  - 0.003
current_goal_constant: 0.001
